pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -98,6 +98,13 @@ default long getNumberOfMessages() {
         default boolean isMarkerMessage() {
             return false;
         }
+
+        default void setProperty(String propertyName, Object value) {
+        }
+
+        default Object getProperty(String propertyName) {
+            return null;
+        }
```

Comment:
Looks not related to this PR?

Reply:
These method are used to store context information from managed ledger interceptors so that they can be made available to the broker interceptors for tracing. For example, when encryption/decryption is done, the time taken for the encryption/decryption can be stored and retrieved using these methods